src/components/message/message-card-list/MessagesVirtualizedList.tsx

```diff
@@ -121,7 +128,10 @@ const MessagesVirtualizedList = (props: Props) => {
 					searchChannelPrev &&
 					!searchChannelPrev.isLoading &&
 					!searchChannelPrev.isEndReached &&
-					(wheelScrollDirection === undefined || wheelScrollDirection === 'previous')
+					!isHorizontal &&
+					((wheelScrollDirection === undefined &&
+						scroller.scrollHeight !== scroller.clientHeight) ||
```

Comment:
why do we need this line ? i think this can lead to a bug when messages will not load.
let's say there no next messages, and all previous messages fitted into screen without overflowing it. in this case scrollHeight === clientHeight and you wont be able to load more previous messages

Reply:
This is for the case with horizontal scroll without wheel, I'll look how to change it